app/helpers/blacklight_helper.rb

```diff
@@ -10,9 +10,11 @@ def document_partial_name(document, base_name = nil)
 
     display_type ||= document.display_type
 
+    display_type ||= document['eds_publication_type']
```

Comment:
The display_type configured in the article controller is `eds_publication_type`, so I'm not quite sure I understand what this is trying to do.